website/docs/migrations/migrate-from-ckb-sdk-core.md

```diff
@@ -0,0 +1,388 @@
+# Migrate From ckb-sdk-core to Lumos
+
+## Summary
+To avoid splitting the development ecosystem, ckb-sdk-js will be merged with lumos, please check the migration list for migration solution. 
```

Comment:
To avoid splitting the development ecosystem, `ckb-sdk-js` will be migrated to `lumos`, please check out the migration list.